packages/flutter_tool_api/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: flutter_tool_api
+description: API for building Flutter tool extensions
+homepage: https://flutter.dev
+author: Flutter Authors <flutter-dev@googlegroups.com>
+
+environment:
+  # The pub client defaults to an <2.0.0 sdk constraint which we need to explicitly overwrite.
+  sdk: ">=2.2.2 <3.0.0"
+
+dependencies:
+  file:
```

Comment:
Is this like 'any' ?

Reply:
any for now, since `update-packages` will diff update everything, I am saving that for the end